open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/DocStoreErrorCode.java

```diff
@@ -0,0 +1,97 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.adapters.adminservices.configurationstore.file;
+
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.ExceptionMessageDefinition;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.ExceptionMessageSet;
+
+/**
+ * The DocStoreErrorCode is used to define first failure data capture (FFDC) for errors that occur when working with
+ * the unencrypted file based doc store.  It is used in conjunction with all Exceptions, both Checked and Runtime (unchecked).
+ *
+ * The 5 fields in the enum are:
+ * <ul>
+ *     <li>HTTP Error Code for translating between REST and JAVA - Typically the numbers used are:</li>
+ *     <li><ul>
+ *         <li>500 - internal error</li>
+ *         <li>400 - invalid parameters</li>
+ *         <li>404 - not found</li>
+ *         <li>409 - data conflict errors - eg item already defined</li>
+ *     </ul></li>
+ *     <li>Error Message Id - to uniquely identify the message</li>
+ *     <li>Error Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>SystemAction - describes the result of the error</li>
+ *     <li>UserAction - describes how a user should correct the error</li>
+ * </ul>
+ */
+public enum DocStoreErrorCode implements ExceptionMessageSet
+{
+    CONFIG_RETRIEVE_ALL_ERROR(400, "UNENCRYPTED-DOC-STORE-400-001 ",
```

Comment:
Again add the directory / file name that is not accessible.  Also a message id prefix such as "FILE-CONFIG-DOC-STORE"
would be much more helpful to identify that is this a configuration document store that uses files - not any old document store.

Reply:
done